.github/workflows/release.crate.yml

```
@@ -0,0 +1,221 @@
+name: release crate
+
+on:
+  push:
+    tags:
+      - "v*"
```

Let's lock SWS to `v2.x` for now.

```suggestion
    tags:
    - 'v2.[0-9]+.[0-9]+'
    - 'v2.[0-9]+.[0-9]+-beta.[0-9]+'
```